### DISCUSSION TITLE
Where is the container of inference-server?

### DISCUSSION THREAD

**Discussion by masa126** [AxisCommunications/acap-computer-vision-sdk-examples#122] [General] [answered]
> **Please do not disclose security vulnerabilities as issues. See our [security policy](../../SECURITY.md) for responsible disclosures.**

In [README.md](https://github.com/AxisCommunications/acap-computer-vision-sdk-examples/tree/main/object-detector-python#readme) , there are three containers for this sample.
The 1st one is built by the Dokerfile in this repository. 
The 3rd one will be created by user.
Where is the 2nd one? 
Is it the docker image of larod-inference-server given by https://hub.docker.com/r/axisecp/larod-inference-server?

**Comment by Corallo** [comment, marked as answer]
Hi @masa126  

As you said, there are three docker images. 
The app and the model container are built by the user, as described in the Readme.
The third one is pulled from docker hub (the link you sent). It should be done automatically by docker compose once the docker compose up command is executed.

I'm going to convert this issue into a discussion, as it is not a bug.

**Reply by masa126** [reply]
I want to try the sample code of object-detector-python on AXIS-P1467.
I built docker with the following commands via ubuntu terminal but failed.
  docker build --build-arg ARCH=aarch64 --build-arg CHIP=artpec8 --tag obj_detectx:1.0 .
  docker cp $(docker create obj_detectx:1.0):/opt/app ./build
![image](https://user-images.githubusercontent.com/50834087/211437537-c3b7357d-ba1a-4114-a92d-030b338fc5e8.png)
Please let me know the way to run this code on P1467 for testing the inference capability of the tflite model.

**Reply by Corallo** [reply]
I think you are mixing up the instructions of the [acap-computer-vision-sdk-example](https://github.com/AxisCommunications/acap-computer-vision-sdk-examples/tree/main/object-detector-python) repository with the [acap-native-sdk-example](https://github.com/AxisCommunications/acap-native-sdk-examples/tree/main/object-detection) repository.
In the computer vision sdk, there is no need to copy the build folder out of the image, you just build the app, and load it in the camera.
In the the native sdk, there is no python support.

Make sure to use the [right instructions for the example](https://github.com/AxisCommunications/acap-computer-vision-sdk-examples/tree/main/object-detector-python#how-to-run-the-code) and also to check the [requirements](https://github.com/AxisCommunications/acap-computer-vision-sdk-examples/tree/main/object-detector-python#how-to-run-the-code) 

Feel free to ask if something is unclear, we appreciate your feedback.
